### PR TITLE
remove obsolete stage filters

### PR DESCRIPTION
**Pull request by KristinBi**
Cleaning up obsolete options in the stage filters including ios, android, mac. 
Also modified the unit test.

Issue Link: https://github.com/flutter/flutter/issues/81062

### CODE REVIEW

File: dashboard/lib/logic/task_grid_filter.dart [hunk 1]
```diff
@@ -87,29 +79,6 @@ class TaskGridFilter extends FilterPropertySource {
...
-  /// by a Windows stage in the devicelab.
-  /// This property will filter out columns on the build dashboard.
-  bool get showWindows => _windowsProperty.value;
-  set showWindows(bool value) => _windowsProperty.value = value;
-
-  /// The [showCirrus] property is a boolean that indicates whether to display tasks produced
-  /// by the Cirrus stage in the devicelab.
-  /// This property will filter out columns on the build dashboard.
-  bool get showCirrus => _cirrusProperty.value;
-  set showCirrus(bool value) => _cirrusProperty.value = value;
```
Comment (CaseyHillers) [inline dashboard/lib/logic/task_grid_filter.dart:112]: You'll need to run https://github.com/flutter/cocoon/blob/main/format.sh to format the code

Or you can add to your `.bashrc`, `alias ff=dart format --set-exit-if-changed --line-length=120 .`

**Review by CaseyHillers — COMMENTED** [review]
LGTM other than the formatting necesary

**Comment by CaseyHillers** [comment]
Please sign the CLA (see checks > cla/google).

> This pull request is not suitable for automatic merging in its current state.

Only add the "waiting for tree to go green" label after getting the necessary approvals on your PR.

**Review by keyonghan — COMMENTED** [review]
Nit: link the issue/bug in the description this PR is trying to fix.

File: dashboard/lib/logic/task_grid_filter.dart (hunk 1 above)
Comment (KristinBi) [reply dashboard/lib/logic/task_grid_filter.dart:112]: Thanks for the instruction. Have changed the format.

**Comment by KristinBi** [comment]
Sorry I'm afraid I have to close this pull request and create another one. When I tried to fix the cla issue, I realized that all the commits should be pushed by the authorized email account to pass the cla check. Since my previous commits used the wrong email, I think the only solution is to recreate a new pull request.

**Pull request closed without merging**